Makefile.am

```diff
@@ -36,195 +36,60 @@ TESTS =
 
 if HAVE_FAPI
 
-bin_PROGRAMS += \
-    tools/fapi/tss2_decrypt \
-    tools/fapi/tss2_encrypt \
-    tools/fapi/tss2_list \
-    tools/fapi/tss2_changeauth \
-    tools/fapi/tss2_delete \
-    tools/fapi/tss2_import \
-    tools/fapi/tss2_getinfo \
-    tools/fapi/tss2_createkey \
-    tools/fapi/tss2_createseal \
-    tools/fapi/tss2_exportkey \
-    tools/fapi/tss2_getcertificate \
-    tools/fapi/tss2_getplatformcertificates \
-    tools/fapi/tss2_gettpmblobs \
-    tools/fapi/tss2_getappdata \
-    tools/fapi/tss2_setappdata \
-    tools/fapi/tss2_setcertificate \
-    tools/fapi/tss2_sign \
-    tools/fapi/tss2_verifysignature \
-    tools/fapi/tss2_verifyquote \
-    tools/fapi/tss2_createnv \
-    tools/fapi/tss2_nvextend \
-    tools/fapi/tss2_nvincrement \
-    tools/fapi/tss2_nvread \
-    tools/fapi/tss2_nvsetbits \
-    tools/fapi/tss2_nvwrite \
-    tools/fapi/tss2_getdescription \
-    tools/fapi/tss2_setdescription \
-    tools/fapi/tss2_pcrextend \
-    tools/fapi/tss2_quote \
-    tools/fapi/tss2_pcrread \
-    tools/fapi/tss2_authorizepolicy \
-    tools/fapi/tss2_exportpolicy \
-    tools/fapi/tss2_import \
-    tools/fapi/tss2_provision \
-    tools/fapi/tss2_getrandom \
-    tools/fapi/tss2_unseal \
-    tools/fapi/tss2_writeauthorizenv
+bin_PROGRAMS += tools/fapi/tss2
 
 endif
 
 noinst_LIBRARIES = $(LIB_COMMON)
 lib_libcommon_a_SOURCES = $(LIB_SRC)
 lib_libcommon_a_CFLAGS = -fPIC $(AM_CFLAGS)
 
-tools_fapi_tss2_decrypt_CFLAGS = $(FAPI_CFLAGS)
-tools_fapi_tss2_decrypt_LDFLAGS = $(TSS2_FAPI_LIBS)
-tools_fapi_tss2_decrypt_SOURCES = tools/fapi/tss2_template.c tools/fapi/tss2_template.h \
-    tools/fapi/tss2_decrypt.c
+tools_fapi_tss2_CFLAGS = $(FAPI_CFLAGS)
```

Comment:
Shouldn't this have:
```C
tools_fapi_CFLAGS = $(AM_CFLAGS) -DTSS2_TOOLS_MAX="$(words $(tss2_tools))"
```

Reply:
Yes, although 1024 tss2 tools should be enough for anyone. Fixed in https://github.com/tpm2-software/tpm2-tools/pull/2123/commits/469e8518f99ddc8c85007d440bf6510d661beac8